src/main/java/edu/harvard/iq/dataverse/dataverse/banners/dto/DataverseLocalizedBannerDto.java

```diff
@@ -32,8 +32,12 @@ public DataverseLocalizedBannerDto(Long id, String locale, byte[] image, String
 
     private StreamedContent miniDisplayImage;
 
+    private StreamedContent uploadedDisplayImage;
```

Comment:
Are you really sure that you need all of these fields in model class.
We have already 3 ones to store image.
And StreamedContent can store contentType and name.